robot/basestation/static/js/pages/rover.js

```diff
@@ -449,7 +444,7 @@ function gameLoop () {
   have the rate (time) at which the commands are actually sent to be
   decoupled from the rate at which the steering or throttle changes
   */
-  if (millisSince(lastCmdSent) > GAME_LOOP_PERIOD) {
```

Comment:
This constant, GAME_LOOP_PERIOD has not be deleted.

Reply:
Deleted the const `GAME_LOOP_PERIOD`.

Reply:
oh baby what is you doing

Reply:
oh baby what is you doing

Reply:
I really don't know what I was doing lol.